tests/many_to_one_null/models.py

```diff
@@ -32,3 +32,18 @@ class Car(models.Model):
 
 class Driver(models.Model):
     car = models.ForeignKey(Car, models.SET_NULL, to_field='make', null=True, related_name='drivers')
+
+
+class Poll(models.Model):
```

Comment:
Is there an issue reusing the `Car`/`Driver` (or some other) models? We prefer not to add new models unless absolutely necessary since that increases complexity and the time needed to run the tests.

Reply:
`Poll` and `Choice` can still be removed. The test using them just requires an unsaved model, so `Car()` triggers the error.

Reply:
should we remove all the Poll and Choice models and related codes?